### DISCUSSION TITLE
Unable to find repo

### DISCUSSION THREAD

**Discussion by melissachang** [community/community#10105] [Other Feature Feedback, Questions, & Ideas]
Here is the the repo I'm trying to find:

https://github.com/DataBiosphere/terra-cli/

When I search for it (https://cs.github.com/?scopeName=All+repos&scope=&q=repo%3ADataBiosphere%2Fterra-cli), I get `Query is not satisfiable`:

![image](https://user-images.githubusercontent.com/10929390/149430297-53318737-b731-4886-a1ca-64fafdccf858.png)

**Comment by look** [comment]
This means that repository hasn't been added yet.

If you know someone who belongs to this organization, getting them to join the Blackbird Technical Preview will index their repositories. Otherwise, I'll run another sweep of public repositories to try to bring this in within the next week or so.

**Update:** I spoke too soon! This repository is indexed but is being excluded. I will look into it more and get it fixed for you. Thanks for the report!

**Reply by octylFractal** [reply]
FYI I experienced the same problem with https://github.com/CerulanLumina/sqpack.
